release/_bom.sh

```diff
@@ -114,6 +129,33 @@ function generate_api_source_jar {
 	done
 }
 
+function generate_distro_jar {
+	download_bnd_files
+
+	lc_cd "${_BUNDLES_DIR}/tomcat/bin"
+
+	./catalina.sh start
+
+	lc_cd "${_BUILD_DIR}/boms"
+
+	chmod u+x biz.aQute.bnd-6.4.0.jar
```

Comment:
@pgydev, why don't you move this command to the function `download_bnd_files` and run it just after the download of this file?

Reply:
I placed it in the `download_bnd_files` and did it like this `chmod u+x "${_BUILD_DIR}/boms/biz.aQute.bnd-6.4.0.jar"`.